src/main/resources/application.yml

```
@@ -29,14 +29,20 @@ spring:
+
 server:
   port: 8081
+
 aes:
   secret-key: ${AES_SECRET}
-  init-vector: ${AES_INIT_VECTOR}
\ No newline at end of file
+  init-vector: ${AES_INIT_VECTOR}
+
+domain:
+  config-path: ${DOMAIN_CONFIG_PATH}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)